docker/incremental_load.sh.tpl

```diff
@@ -24,7 +24,7 @@ DOCKER="${DOCKER:-docker}"
 
 # Fetch the diff ids of the layers loaded in the docker daemon already
 IMAGES=$(${DOCKER} inspect $(${DOCKER} images -aq) |& \
-  grep sha256 | grep -v Id | cut -d'"' -f 2 | cut -d':' -f 2 | \
+  grep -E '^ +"sha256:' | grep -v Id | cut -d'"' -f 2 | cut -d':' -f 2 | \
```

Comment:
Yeah, I'd love to depend on `jq`, but I think that's a non-starter for some of the environments this needs to work in :(